### DISCUSSION TITLE
v0 local deployment

### DISCUSSION THREAD

**Discussion by ngoiyaeric** [vercel/community#6580] [Help]
### Summary

Hi, I'm having a hard time deploying my v0 page through the latest @ id cli command. When I run npx run dev a generic nextjs landing page pops up asking me to link components in the src/app file but I've already ported the components through the terminal command. 

### Steps to Reproduce

https://v0.dev/docs#adding-v0-components 

These steps here

**Comment by ngoiyaeric** [comment]
![Uploading Screen Shot 2024-04-21 at 10.28.51 AM.png…]()

**Comment by amyegan** [comment]
Hey @ngoiyaeric. I wasn't able to replicate this behavior just now. If it's still a problem for you, please share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) so I can take a closer look 🙏
